### PR TITLE
feat: add Gaudi 2 icon

### PR DESCRIPTION
**Pull request by agatha197**
- agent summary

- agent

- session list

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [x] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by agatha197** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2630?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2630** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2630?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2623** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2623?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @agatha197 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: react/src/components/ResourceNumber.tsx [hunk 1]
```diff
@@ -130,7 +130,7 @@ export const ResourceTypeIcon: React.FC<AccTypeIconProps> = ({
     'tpu.device': [<MWCIconWrap size={size}>view_module</MWCIconWrap>, 'TPU'],
     'ipu.device': [<MWCIconWrap size={size}>view_module</MWCIconWrap>, 'IPU'],
     'atom.device': ['/resources/icons/rebel.svg', 'ATOM'],
-    'gaudi2.device': ['/resources/icons/npu_generic.svg', 'Gaudi 2'],
+    'gaudi2.device': ['/resources/icons/intel-gaudi.svg', 'Gaudi 2'],
```
Comment (rapsealk) [inline react/src/components/ResourceNumber.tsx]:
```suggestion
    'gaudi2.device': ['/resources/icons/Intel-gaudi.svg', 'Gaudi 2'],
```

File: src/components/backend-ai-agent-summary-list.ts [hunk 2]
```diff
@@ -755,7 +758,7 @@ export default class BackendAIAgentSummaryList extends BackendAIPage {
                   <div class="layout horizontal start resource-indicator">
                     <img
                       class="indicator-icon fg green"
-                      src="/resources/icons/npu_generic.svg"
+                      src="/resources/icons/intel-gaudi.svg"
```
Comment (rapsealk) [inline src/components/backend-ai-agent-summary-list.ts]:
```suggestion
                      src="/resources/icons/Intel-gaudi.svg"
```

File: src/components/backend-ai-session-list.ts [hunk 3]
```diff
@@ -3519,7 +3522,7 @@ ${rowData.item[this.sessionNameField]}</pre
                 ? html`
                     <img
                       class="indicator-icon fg green"
-                      src="/resources/icons/npu_generic.svg"
+                      src="/resources/icons/intel-gaudi.svg"
```
Comment (rapsealk) [inline src/components/backend-ai-session-list.ts]:
```suggestion
                      src="/resources/icons/Intel-gaudi.svg"
```

File: resources/icons/Intel-gaudi.svg
Comment (rapsealk) [inline resources/icons/Intel-gaudi.svg:1]: Please check the file name: **I**ntel-gaudi.svg.

**Review by rapsealk — APPROVED** [review]
LGTM!

**Review by yomybaby — APPROVED** [review]
LGTM

**Pull request merged**